src/utils/getFilteredList.ts

```diff
@@ -0,0 +1,21 @@
+import { TodoFilter } from '../types/TodoFilter';
+import { Todo } from '../types/Todo';
+
+export const getFilteredList = (
+  currentFilter: TodoFilter,
+  todosList: Todo[] | null,
+): Todo[] | null => {
+  if (currentFilter === 'All') {
```

Comment:
use enum value instead of magic string
```suggestion
  if (currentFilter === TodoFilter .all) {
```